pkg/bff/models/v1/registration.go

```diff
@@ -550,39 +550,175 @@ func (r *RegistrationForm) ValidateContacts() error {
+	err := make(ValidationErrors, 0)
 	name := strings.TrimSpace(contact.Name)
 	if name == "" {
-		v = append(v, &ValidationError{Field: fieldName + ".name", Err: ErrMissingField.Error()})
+		err = append(err, &ValidationError{Field: fieldName + ".name", Err: ErrMissingField.Error()})
 	} else if len(name) < 2 {
 		// Check that the name is long enough to match GDS validation
-		v = append(v, &ValidationError{Field: fieldName + ".name", Err: ErrTooShort.Error()})
+		err = append(err, &ValidationError{Field: fieldName + ".name", Err: ErrTooShort.Error()})
 	}
 
 	email := strings.TrimSpace(contact.Email)
 	if email == "" {
-		v = append(v, &ValidationError{Field: fieldName + ".email", Err: ErrMissingField.Error()})
+		err = append(err, &ValidationError{Field: fieldName + ".email", Err: ErrMissingField.Error()})
 	} else {
 		// Check that the email is parseable by RFC 5322.
-		if _, err := mail.ParseAddress(email); err != nil {
-			v = append(v, &ValidationError{Field: fieldName + ".email", Err: ErrInvalidEmail.Error()})
+		if _, verr := mail.ParseAddress(email); verr != nil {
+			err = append(err, &ValidationError{Field: fieldName + ".email", Err: ErrInvalidEmail.Error()})
 		}
 	}
 
 	phone := strings.TrimSpace(contact.Phone)
 	if phone == "" {
-		v = append(v, &ValidationError{Field: fieldName + ".phone", Err: ErrMissingField.Error()})
+		err = append(err, &ValidationError{Field: fieldName + ".phone", Err: ErrMissingField.Error()})
 	}
 
 	// TODO: Ensure this is a valid phone number
 
-	return v
+	if len(err) == 0 {
+		return nil
+	}
+
+	return err
 }
 
 // Validate only the fields in the trixo step.
 func (r *RegistrationForm) ValidateTRIXO() error {
-	// TODO: implement
-	return nil
+	err := make(ValidationErrors, 0)
+
+	if r.Trixo == nil {
+		err = append(err, &ValidationError{
+			Field: FieldTRIXO,
+			Err:   ErrMissingField.Error(),
+		})
+		return err
+	}
+
+	// TODO: Validate country name or ISO-3166-1 code
+	if strings.TrimSpace(r.Trixo.PrimaryNationalJurisdiction) == "" {
+		err = append(err, &ValidationError{
+			Field: FieldTRIXOPrimaryNationalJurisdiction,
+			Err:   ErrMissingField.Error(),
+		})
+	}
+
+	if strings.TrimSpace(r.Trixo.PrimaryRegulator) == "" {
+		err = append(err, &ValidationError{
+			Field: FieldTRIXOPrimaryRegulator,
+			Err:   ErrMissingField.Error(),
+		})
+	}
+
+	finTransfers := strings.ToLower(strings.TrimSpace(r.Trixo.FinancialTransfersPermitted))
+	if finTransfers == "" {
+		err = append(err, &ValidationError{
+			Field: FieldTRIXOFinancialTransfersPermitted,
+			Err:   ErrMissingField.Error(),
+		})
+	} else if finTransfers != "yes" && finTransfers != "no" && finTransfers != "partially" {
+		err = append(err, &ValidationError{
+			Field: FieldTRIXOFinancialTransfersPermitted,
+			Err:   ErrInvalidField.Error(),
+		})
+	}
+
+	for i, juris := range r.Trixo.OtherJurisdictions {
+		// TODO: Validate country name or ISO-3166-1 code
+		if strings.TrimSpace(juris.Country) == "" {
```

Comment:
I've just been checking that Country is 2 characters and using `ErrInvalidCountry` if it's not for right now.